_data/cryptocurrencies.yml

```
@@ -6,6 +6,13 @@ websites:
       - totp
     doc: https://support.anycoindirect.eu/hc/en-gb/articles/115001423191
 
+  - name: Bcnex
+    url: https://www.bcnex.net/
+    img: bcnex.png
+    tfa:
+      - totp
```

The doc link also shows SMS-based 2FA.
```suggestion
      - totp
      - sms
```